src/distPlus/distribution/Continuous.re

```diff
@@ -205,23 +248,34 @@ let combineAlgebraicallyWithDiscrete =
       t1: t,
```

Comment:
I think that combineAlgebraicallyWithDiscrete should probably go in the ``Shape`` file, in part, because there's no clear reason why this should not be in the discrete file instead (there called, ``combineAlgebraicallyWithContinuous``.

Reply:
I'd prefer that too, but the function is also called by the Mixed module, and so putting it in Shape creates a dependency cycle. We could consider putting it in the Mixed file, but I think that would lead to even more confusion. The reason it's in Continuous right now is because it returns a continuousShape. I'm very open to suggestions though as it's clearly an awkward solution

Reply:
Got it. I'll think about this, it seems tricky. Dependency cycles can be challenging, the whole Mixed/Continuous/Discrete combo is kind of awkward for that reason.